visualization/resources/web/vis/genericChart/genericChartHelper.js

```diff
@@ -1450,8 +1451,14 @@ LABKEY.vis.GenericChartHelper = new function(){
             });
 
             var wrapLines = _wrapXAxisTickTextLines(scales, plotConfig, maxLen, data);
-            // min bottom margin: 50, max bottom margin: 150
-            margins.bottom = Math.min(150, 60 + ((wrapLines - 1) * 25));
+
+            if (!chartConfig.legendPos || chartConfig.legendPos === 'right') {
+                // min bottom margin: 60, max bottom margin: 150
+                margins.bottom = Math.min(150, 60 + ((wrapLines - 1) * 25));
+            } else {
+                // min bottom margin: 166, max bottom margin: 360
+                margins.bottom = Math.min(360, 170 + ((wrapLines - 1) * 25));
+            }

```

Comment:
I was testing this locally by setting legendPos = 'none' for all of my charts and it is going into this else statement and adding the extra bottom margin when it shouldn't be. I think we want the 'none' case to go into the if statement block, right? If so, maybe just flip this so that:
```suggestion
            if (chartConfig.legendPos === 'bottom') {
                // min bottom margin: 166, max bottom margin: 360
                margins.bottom = Math.min(360, 170 + ((wrapLines - 1) * 25));
            } else {
                // min bottom margin: 60, max bottom margin: 150
                margins.bottom = Math.min(150, 60 + ((wrapLines - 1) * 25));
            }
```